src/screens/HomePage/Announcements/Announcements.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import {
+  AnnouncementsSectionHeader,
+  AnnouncementIntro,
+  Wrapper,
+  Date,
+  AnnouncementBodyCopy,
+  ButtonsContainer,
+  ReadMoreButton,
+  ViewAllLink,
+} from './Announcements.style';
+
+const Announcements = () => {
+  return (
+    <Wrapper>
+      <AnnouncementsSectionHeader>Announcements</AnnouncementsSectionHeader>
+      <AnnouncementIntro>
+        Introducing our newest key indicator, “New infections per capita”
+      </AnnouncementIntro>
+      <Date>Tuesday, July 1, 2020</Date>
+      <AnnouncementBodyCopy>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+        mollit anim id est laborum.
+      </AnnouncementBodyCopy>
+      <ButtonsContainer>
+        <ReadMoreButton>Read more</ReadMoreButton>
+        <ViewAllLink href="/">View all announcements</ViewAllLink>
+      </ButtonsContainer>
```

Comment:
yeah, "/" is just place-holder for the time being

Reply:
both now go to the blog! ReadMoreButton's href will be updated with the specific blog post

Reply:
One more, if the link is supposed to open in a new tab, you need to add `rel="noopener"` to avoid security and performance issues, see https://web.dev/external-anchors-use-rel-noopener/